Billing.API.Test/InvoiceApiTest.cs

```diff
@@ -346,7 +346,7 @@ public async Task GetInvoices_WhenNoParameters_ReturnsNotFound()
         }
 
         [Theory]
-        [InlineData("accounts/doppler/1/invoices/invoice_2020-01-01_123.pdf?_s=sOvP2buZIW8IFDAXMIe8BROG9GoB7zLPCzAv3OzVs")]
+        [InlineData("accounts/doppler/1/invoices/invoice_AR_2020-01-01_123.pdf?_s=6RTlC0JYyqMTj1LaHAbnTWQFLC9feeUNSNzzkRlE")]
```

Comment:
Que opinás de dejar ambas URLs? las dos deberían funcionar

```suggestion
        [InlineData("accounts/doppler/1/invoices/invoice_2020-01-01_123.pdf?_s=sOvP2buZIW8IFDAXMIe8BROG9GoB7zLPCzAv3OzVs")]
        [InlineData("accounts/doppler/1/invoices/invoice_AR_2020-01-01_123.pdf?_s=6RTlC0JYyqMTj1LaHAbnTWQFLC9feeUNSNzzkRlE")]
```

Reply:
yes, it is correct. I will add it.

Reply:
Done